config/config.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017 Uber Technologies, Inc.
+// Copyright (c) 2017-2018 Uber Technologies, Inc.
```

Comment:
Yes lol talk to @yurishkuro.

Reply:
seems painful... I'll see how we can automate this

Reply:
I think it can be done with the existing script if you append `-<current-year>` and limit the files to the list given by this command `git diff --name-status master | grep -E -v '^M' | sed 's/^M\s\+//g'`. That looks scarier than it is ;).

Reply:
This doesn't seem to work on my Mac. Unclear why not.

Reply:
Spent way too long researching this... Mac chokes on tab characters (see tab here https://github.com/git/git/blob/c6284da4ff4afbde8211efe5d03f3604b1c6b9d6/combine-diff.c#L1185). Maybe `git diff --name-status master | grep -E '^M' | tr '\t' ';' |  sed 's/^M;//g'` instead.

Reply:
> Mac chokes on tab characters

with what error?

Reply:
Actually more bizarre than that. In my terminal, any attempt to evaluate a tab character causes autocomplete. If I copy and paste this `M       process.go` from the output into my input I get `Makefileprocess.go`. There seems to be no way to simply sed with a tab character on my Mac.